include/hpp/core/config-validation.hh

```diff
@@ -75,6 +75,11 @@ namespace hpp {
       virtual void filterCollisionPairs (const RelativeMotion::matrix_type&)
       {}
 
+      /// Rearrange the collisions pairs of all configValidation in a random manner
+      /// \brief randomnizeCollisionPairs
+      ///
+      virtual void randomnizeCollisionPairs(){}
```

Comment:
There is a typo here.

Reply:
Actually this method is not used anymore since I implemented https://github.com/humanoid-path-planner/hpp-core/commit/7f5d408a013ca9aeb0933dfa69da5993c8093453 .
I thought I removed it ... Sorry about that.